wave_controls/src/pid.cpp

```diff
@@ -0,0 +1,51 @@
+#include "wave/controls/pid.hpp"
+
+namespace wave {
+
+PID::PID(void) {
+    this->error_prev = 0.0;
+    this->error_sum = 0.0;
+
+    this->error_p = 0.0;
+    this->error_i = 0.0;
+    this->error_d = 0.0;
+
+    this->k_p = 0.0;
+    this->k_i = 0.0;
+    this->k_d = 0.0;
+}
+
+PID::PID(double k_p, double k_i, double k_d) {
+    this->error_prev = 0.0;
+    this->error_sum = 0.0;
+
+    this->error_p = 0.0;
+    this->error_i = 0.0;
+    this->error_d = 0.0;
+
+    this->k_p = k_p;
```

Comment:
You're right there is no difference. It would only be for consistency with cases where initializer lists are required.